pkg/booleanpolicy/violationmessages/printer/container.go

```diff
@@ -169,10 +169,33 @@ func imageSignatureVerifiedPrinter(fieldMap map[string][]string) ([]string, erro
 		ContainerName: maybeGetSingleValueFromFieldMap(augmentedobjs.ContainerNameCustomTag, fieldMap),
 		Status:        "unverified",
 	}
-	if verifiedBy := maybeGetSingleValueFromFieldMap(augmentedobjs.ImageSignatureVerifiedCustomTag, fieldMap); verifiedBy != "" {
-		r.Status = "verified by " + verifiedBy
+
+	tmpl, err := getTemplate(imageSignatureVerifiedTemplate)
+	if err != nil {
+		return nil, err
 	}
-	return executeTemplate(imageSignatureVerifiedTemplate, r)
+
+	var result []string
+	if ids, ok := fieldMap[augmentedobjs.ImageSignatureVerifiedCustomTag]; ok {
+		for _, id := range ids {
+			if id != "" && id != "<empty>" {
```

Comment:
Can `<empty>` be returned as one of ids here?

Reply:
Yes, this is why this check.